test/factories/events.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :event do
+    title { "MyString" }
```

Comment:
Why is everything hardcoded in the factories?

Reply:
They are now all generating unique/random/custom data.